torchtune/data/_collate.py

```diff
@@ -81,13 +81,14 @@ def padded_collate(
             padding values.
 
     Returns:
-        torch.Tensor: The padded tensor of input ids with shape [batch_size, max_seq_len].
+        torch.Tensor: The padded tensor of input ids with shape ``[batch_size, max_seq_len]``.
 
     Raises:
-        ValueError: if ``pad_direction`` is not one of "left" or "right".
-        ValueError: if ``keys_to_pad`` is empty, or is not a list, or is not a subset of keys in the batch.
-        ValueError: if ``padding_idx`` is provided as a dictionary, but the keys are not identical to
-            ``keys_to_pad``.
+        ValueError:
```

Comment:
The docstring linter lints based on *unique* raises, so we document a single value error and each condition it will raise for. Annoyingly, I couldn't for the life of me get lists to properly render in the raises section.

This is what it looks like instead:

<img width="1656" alt="image" src="https://github.com/user-attachments/assets/5f4874be-6c4e-41de-9480-4275f48ac12d" />